docs/pt/week07/07.md

```diff
@@ -0,0 +1,24 @@
+---
+lang-ref: ch.07
+title: Semana 7
+lang: pt
+translation-date: 5 March 2021
+translator: Catarina Carvalho
+---
+<!--## Lecture part A
+-->
+## Aula - parte A
```

Comment:
Elsewhere https://github.com/Atcold/pytorch-Deep-Learning/commit/cdc4ab04d5c7e3481c5288a48074f2f2eeb7739f#diff-4df956a989acc43055011b7087373cde3ed76f6d15d5b7733ed341512e6ccdf1R15 "Curso" has been used instead of "Aula". We should be consistent. I don't know which translation is better.

Reply:
I think this is a difference between PT-BR and PT-PT. In any case, it makes sense to continue with the previous notation 👍 . I will fix it.